database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Views/ContentView.swift

```diff
@@ -18,7 +18,7 @@ import SwiftUI
 import Firebase
 
 struct ContentView: View {
-  @AppStorage("isSignedIn") var isSignedIn = false
+  @AppStorage("isSignedIn") var isSignedIn = true
```

Comment:
How is this determined / what's the motivation behind this change?

Reply:
The declaration of isSignedIn variable here is not required. The purpose for the variable value is mainly used for testing purposes. Initially when I was implementing for login/logout it is more convenient to be logged out as the initial state, and for making and fetching post it is more convenient to be logged in as the initial state.

Reply:
sgtm